clients/python/Makefile

```diff
@@ -16,22 +16,39 @@ clean:
 .PHONY: deploy-latest-mr
 deploy-latest-mr:
 	cd ../../ && IMG_VERSION=${IMG_VERSION} make image/build ARGS=--load && LOCAL=1 ./scripts/deploy_on_kind.sh
-	kubectl port-forward -n kubeflow services/model-registry-service 8080:8080 &
+	kubectl port-forward -n kubeflow services/model-registry-service 8080:8080 & echo $$! >> .port-forwards.pid
 
 .PHONY: deploy-test-minio
 deploy-test-minio:
 	cd ../../ && ./scripts/deploy_minio_on_kind.sh
-	kubectl port-forward -n minio svc/minio 9000:9000 &
+	kubectl port-forward -n minio svc/minio 9000:9000 & echo $$! >> .port-forwards.pid
 
 .PHONY: deploy-local-registry
 deploy-local-registry:
 	cd ../../ && ./scripts/deploy_local_kind_registry.sh
+	kubectl port-forward service/distribution-registry-test-service 5001:5001 & echo $$! >> .port-forwards.pid
 
 .PHONY: test-e2e
 test-e2e: deploy-latest-mr deploy-local-registry deploy-test-minio
+	@echo "Starting test-e2e"
+	-$(MAKE) test-e2e-run; STATUS=$$?
```

Comment:
Note the `-` in the `-$(MAKE)` command. This will ensure the `test-e2e` make command will not exit if the sub-command `test-e2e-run` fails. Instead the status is captured and re-used for the `@exit` below